Find a functionally equivalent code:com.android.internal.os.ZygoteConnection.handleParentProc:COMMENT
Method Modifier: private     internal    
Comment:/**
 * Handles post-fork cleanup of parent proc
 *
 * @param pid != 0; pid of child if &gt; 0 or indication of failed fork
 * if &lt; 0;
 * @param descriptors null-ok; file descriptors for child's new stdio if
 * specified.
 * @param pipeFd null-ok; pipe for communication with child.
 * @param parsedArgs non-null; zygote args
 * @return true for "exit command loop" and false for "continue command
 * loop"
 */

@@ -40,46 +40,46 @@
         mSocketOutStream.writeInt(pid);
         mSocketOutStream.writeBoolean(usingWrapper);
     } catch (IOException ex) {
-        Log.e(TAG, "Error reading from command socket", ex);
+        Log.e(TAG, "Error writing to command socket", ex);
         return true;
     }
     return false;

